web/src/components/SpeakerCard.vue

```diff
@@ -44,24 +44,27 @@ export default {
       type: Object,
       required: true,
     },
+    prefectures: {
+      type: Array,
+      required: true,
+    },
   },
   data: () => ({
-    topics: [],
   }),
   computed: {
     jobTitle() {
-      if (this.speaker.job_title && this.speaker.company) {
-        return `${this.speaker.job_title} at ${this.speaker.company}`;
+      if (this.speaker.get('job_title') && this.speaker.get('company')) {
+        return `${this.speaker.get('job_title')} at ${this.speaker.get('company')}`;
       }
-      return this.speaker.job_title || this.speaker.company;
+      return this.speaker.get('job_title') || this.speaker.get('company');
     },
-  },
-  mounted() {
-    this.getTopics();
-  },
-  methods: {
-    getTopics() {
-      this.topics = this.speaker.topics.split(', ');
+    prefecture() {
+      const locationId = this.speaker.fields.location_id[0];
+      const location = this.prefectures.find((elem) => (elem.id === locationId));
+      return location?.fields?.prefecture;
```

Comment:
I did not know about this syntax because I learned javascript through osmosis. Does it just check that they are populated?

Reply:
yeah, it's just a safety check for `null` to avoid a bunch of errors trying to access properties on `null` while it's not populated yet